packages/utility/agent.ts

```diff
@@ -82,7 +82,7 @@ const requests = {
   patch: <T>(url: string, body: {}) =>
     axios.patch<T>(url, body).then(responseBody),
   delete: <T>(url: string, body?: {}) =>
-    axios.delete<T>(url, body).then(responseBody),
+    axios.delete<T>(url, { data: body }).then(responseBody),
```

Comment:
I think this is because strapi added the data field when we migrated from v3 to v4. Should we use an empty object if body is empty though? I think this code will send `{"data":{}}` instead of `{}`. 
Let's do it in another PR: does that mean put and patch should also wrap the body under `data`?

Reply:
@chungthuang I discussed this with Mohammed on a previous pull request (https://github.com/dev-launchers/dev-launchers-platform/pull/1894)

To summarize, the body could not reach the backend before or after this change, but I left it like this because I found a thread that said the way we have implemented previously is wrong-https://stackoverflow.com/questions/51069552/axios-delete-request-with-request-body-and-headers

Mohammed proposed upgrading Axios to solve this issue. I think that is outside the scope of this ticket as we are able to send the DELETE requests with expected results now.